src/Aspire.Dashboard/Components/Pages/ComponentTelemetryContext.cs

```diff
@@ -32,7 +34,7 @@ public async Task InitializeAsync(DashboardTelemetryService telemetryService)
             });
     }
 
-    public bool UpdateTelemetryProperties(ReadOnlySpan<ComponentTelemetryProperty> modifiedProperties)
+    public bool UpdateTelemetryProperties(IEnumerable<ComponentTelemetryProperty> modifiedProperties)
```

Comment:
FYI this change will cause an allocation when doing `UpdateTelemetryProperties([...])`